Cookbook/Technical-Documents/Localizations.md

```diff
@@ -0,0 +1,59 @@
+# How to deal with Localizations [draft]
+
+In project we can add a new Localization. But that is not in target level. Babylon needs to have a different sets of languages/locales per product.
+
+Babylon:
+  - British English
+  - Arabic (soon), etc 
+
+NHS111 and Bupa:
+  - British English
+
+Telus:
+  - Canadian English
+  - Canadian French (soon)
+
+Babylon US:
+  - American English
+  - American Spanish
+
+## Build Settings
+### Understanding `app_languages` variable
+Given that, we manage our languages in an array in Build Settings in each target.
+
+| Babylon UK | Babylon US | Telus |
+|--------|--------|--------|
+| `en.lproj/*` | `en-US.lproj/*` `es-US.lproj/*` | `en.lproj/*` |
+
+**I.E we control the languages for each target editing the `APP_LANGUAGES` under _User-Defined_**
+
+### _Excluded Source File Names_ and _Included Source File Names_ under `Build option`
+In the _Excluded Source File Names_ we exclude any language (`.lproj/`) previously set:
+`*.lproj/*` 
+
+In the _Included Source File Names_ we include the languages (`.lproj/`) set in the above mentioned `app_languages` array:
+`${APP_LANGUAGES}` in Babylon US for example it is en-US.lproj/* es-US.lproj/*
```

Comment:
I don't fully understand what this is. 😞

Reply:
The full content will explain in details. With Xcode screenshots.

Reply:
<img width="974" alt="Screen Shot 2019-07-05 at 14 39 53" src="https://user-images.githubusercontent.com/17405823/60726183-df986b00-9f32-11e9-86da-bd96e2fd52f3.png">

With the target Babylon US selected, on Build settings / Build options / Include Source File Names, the value of the  variable ${APP_LANGUAGES} is shown: `en-US.lproj/* es-US.lproj/*` **space separated**. 

Is it clear? 👍

Reply:
in comments I can drag 'n drop images. In the `.md` I need to host them.

Reply:
To clarify, by default all `lproj` files are excluded aside from the ones in _Include Source File Names_?

Reply:
@j531 yes, we exclude anything and re add the specified ones in the array.